Effect: Fix failure to start using Flatpak-preferred paths
Author: Stephan Sokolow
License: CC0-1.0

--- tomatoes-1.55/makefile	2004-11-20 13:50:09.000000000 -0500
+++ makefile.new	2022-09-06 20:35:24.992375677 -0400
@@ -14,21 +14,21 @@
 # necessary. Remember to include the trailing /

 # MPK directory (where 'tomatoes.mpk' is), default: ./
-MPKDIR = ./
+MPKDIR = /app/share/tomatoes/

 # Music directory (where the music files are), default: ./music/
-MUSICDIR = ./music/
+MUSICDIR = /app/share/tomatoes/music/

 # Hiscore directory (where the hiscores are written to), default: ./
 # We need read/write access!
-HISCOREDIR = ./
+HISCOREDIR = /app/share/tomatoes/

 # Config directory (where the 'config.cfg' is), default: ./
 # We need read/write access!
-CONFIGDIR = ./
+CONFIGDIR = /app/share/tomatoes/

 # Override directory (unused at the moment), default: ./data/
-OVERRIDEDIR = ./data/
+OVERRIDEDIR = /app/share/tomatoes/data/


 DIR_DEFINES = -DLINUX -DMPK_DIR=\"$(MPKDIR)\" -DMUSIC_DIR=\"$(MUSICDIR)\" -DHISCORE_DIR=\"$(HISCOREDIR)\" -DCONFIG_DIR=\"$(CONFIGDIR)\" -DOVERRIDE_DIR=\"$(OVERRIDEDIR)\"
